src/coreclr/jit/llvm.cpp

```diff
@@ -1387,6 +1405,20 @@ void Llvm::buildCnsLng(GenTree* node)
     mapGenTreeToValue(node, _builder.getInt64(node->AsLngCon()->LngValue()));
 }
 
+void Llvm::buildFieldList(GenTreeFieldList* fieldList)
```

Comment:
I think this should be straightforward to implement? Store the individual fields into the `alloca`, then load the result.